vpr/src/util/vpr_utils.cpp

```diff
@@ -523,39 +517,50 @@ bool is_empty_type(t_logical_block_type_ptr type) {
     return type == device_ctx.EMPTY_LOGICAL_BLOCK_TYPE;
 }
 
-t_physical_tile_type_ptr physical_tile_type(ClusterBlockId blk) {
-    auto& place_ctx = g_vpr_ctx.placement();
+t_physical_tile_type_ptr physical_tile_type(t_pl_loc loc) {
     auto& device_ctx = g_vpr_ctx.device();
 
-    auto block_loc = place_ctx.block_locs[blk].loc;
-
-    return device_ctx.grid.get_physical_type({block_loc.x, block_loc.y, block_loc.layer});
+    return device_ctx.grid.get_physical_type({loc.x, loc.y, loc.layer});
 }
 
+//t_physical_tile_type_ptr physical_tile_type(ClusterBlockId blk) {
+//    auto& place_ctx = g_vpr_ctx.placement();
+//    auto& device_ctx = g_vpr_ctx.device();
+//
+//    auto block_loc = place_ctx.block_locs[blk].loc;
+//
+//    return device_ctx.grid.get_physical_type({block_loc.x, block_loc.y, block_loc.layer});
+//}
```

Comment:
Did you intend on leaving this code commented in? I think this should be deleted.